doc/template/HOW-TO.md

```diff
@@ -0,0 +1,128 @@
+## Setup
+
+I start cloning our example
+
+    $ git clone -b gh-pages https://github.com/swcarpentry/lesson-example.git
```

Comment:
`gh-pages` is the default branch for that repository, so you can drop the `-b gh-pages`.